apps/web/src/app/(main)/dashboard/newsletter/[id]/page.tsx

```
@@ -0,0 +1,60 @@
+                    {post.tags.map((tag) => (
+                        <span
+                            key={tag}
+                            className="bg-muted text-muted-foreground border border-border-primary px-3 py-1 text-sm rounded-full"
+                        >
+                            #{tag}
+                        </span>
+                    ))}
+                </div>
+                <article
+                    className="rich-text"
+                    dangerouslySetInnerHTML={{ __html: addLazyLoading(post.content) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)